cupy/cuda/cufft.pyx

```diff
@@ -236,6 +237,7 @@ class PlanNd(object):
         self.fft_type = fft_type
         self.plan = plan
         self.work_area = work_area
+        self.batch = batch
```

Comment:
This line is added solely for bookkeeping purpose and, unlike its `Plan1d` counterpart, is not used in the current implementation, but I envision it might be useful for performing some checks.

Reply:
Is this just to be consistent with the attribute present for the 1D plan? I'm not sure I would add it here until/unless it is needed, but let's see what the core cupy devs think.

Reply:
Yeah it's mostly useful when debugging the CuPy code, probably not important to users.

Reply:
In what debug cases is this attribute useful for us?

Reply:
When one wanna heck if the batch size is set correctly with respect to the axes to be transformed.

Reply:
*check, sorry

Reply:
You can still check the batch size without this attribute, and it's not worth making it a public attribute for the sake of debugging convenience.

Reply:
ok I removed it.